test/functional/wallet_groups.py

```diff
@@ -17,9 +17,17 @@
 class WalletGroupTest(BitcoinTestFramework):
     def set_test_params(self):
         self.setup_clean_chain = True
-        self.num_nodes = 4
+        self.num_nodes = 5
         self.extra_args = [
-            [], [], ['-avoidpartialspends'], ["-maxapsfee=0.01"]]
+            [],
+            [],
+            ['-avoidpartialspends'],
+            # 2.93 XPI is the threshold that causes a node to prefer a
+            # non-grouped tx (3 inputs, 2 outputs) to a grouped tx (5 inputs,
+            # 2 outputs). The fee for the grouped tx is 294 sats higher than
+            # the fee for the non-grouped tx. See tx5 below.
```

Comment:
It seems like this should be 0.000293 XPI, given the comment is talking about 294 sats